### DISCUSSION TITLE
Why does Sumatra appear twice in Control Panel?

### DISCUSSION THREAD

**Discussion by Paultx** [sumatrapdfreader/sumatrapdf#3027] [General]
When I go to **Control Panel > Uninstall or change a program** I get Sumatra PDF listed twice, with the same version but different dates.

![Sumatra_WTF_01](https://user-images.githubusercontent.com/71100640/196221842-b9f9691f-2841-4796-9924-9796b6c0ad5f.png)

Why is that? TIA.

System
Windows 10 Pro 21H1 19043.2130

**Comment by GitHubRulesOK** [comment]
hmm its usually if i installed two different ways like personal and system
best to uninstall both if possible in reverse order so 06/07 1st and clear older 30/06 if possible
then reinstall only one
